Parse/PFAnalytics.m

```diff
@@ -25,9 +25,11 @@ @implementation PFAnalytics
 ///--------------------------------------
 
 + (BFTask *)trackAppOpenedWithLaunchOptions:(NSDictionary *)launchOptions {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_WATCH
+    NSDictionary *userInfo = nil;
+#elif TARGET_OS_IOS
     NSDictionary *userInfo = launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
-#else
+#elif TARGET_OS_MAC
```

Comment:
We can do it that way, sure. I mean, we're still screwed when apple releases a new platform, but I think it's a bit less misleading that way.